dlt/helpers/streamlit_helper.py

```diff
@@ -225,16 +227,28 @@ def _query_data(query: str, chunk_size: int = None) -> pd.DataFrame:
         except SqlClientNotAvailable:
             st.error("Cannot load data - SqlClient not available")
 
-
-    if schema_name:
-        schema = pipeline.schemas[schema_name]
-    else:
-        schema = pipeline.default_schema
-    st.title(f"Available tables in {schema.name} schema")
-
-    for table in schema.data_tables():
+    # Not needed anymore(?)
+    # if schema_name:
+    #     schema = pipeline.schemas[schema_name]
+    # else:
+    #     schema = pipeline.default_schema
```

Comment:
Yes, it's not needed anymore. Please remove it.

Reply:
Done.